config/liota.conf

```diff
@@ -10,6 +10,9 @@ entity_file_path = /usr/lib/liota/entity
 iotcc_path = /usr/lib/liota/iotcc.json
 iotcc_load_retry = 3
 
+[CRL_PATH]
```

Comment:
@gkmccready I have made this change after your approval. I believe it should be correct to set the conf value "blank" as this is the preferred way in python rather than passing None which will be required to be parsed as string in the code.

https://docs.python.org/3/library/configparser.html#supported-ini-file-structure
section [No Values]

Reply:
Are "" because they've been declared but no value assigned? versus None if they're not listed at all? I was just hoping not to test ("" || None) everywhere in the code...

Reply:
I have tested "" works. But I don`t want user to set the value "/etc/liota/mqtt/cert.cl" later as the conf file parser will not accept quotes so I have set it as blank.

Reply:
If it works. I'm not a Python guru so I'm not 100% sure where we should be using "" vs None. I just don't like testing against both everywhere. You lost me on the quoted filename?